packages/offix-datastore/tests/DataStore.test.ts

```diff
@@ -22,11 +19,32 @@ function getIndexedDB() {
     });
 }
 
+interface Note {
```

Comment:
So users can define those or use codegen to generate it :D

Reply:
Yeah.